sktime/forecasting/var.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+__all__ = ["VectorAutoRegression"]
+__author__ = ["Taiwo Owoseni"]
+
+from statsmodels.tsa.api import VAR as _VAR
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+from sktime.utils.validation.series import check_series
+from sktime.utils.validation.forecasting import check_X
+import pandas as pd
+import numpy as np
+
+DEFAULT_ALPHA = 0.05
+
+
+class VectorAutoRegression(_StatsModelsAdapter):
+    _fitted_param_names = ("aic", "fpe", "hqic", "bic")
+    _tags = {
+        "univariate-only": False,
+        "requires-fh-in-fit": False,
+        "handles-missing-data": False,
+    }
+
+    def __init__(self, dates=None, freq=None, missing="none"):
+        # Model params
+        self._y = None
+        self.dates = dates
+        self.freq = freq
+        self.missing = missing
+
+        super(VectorAutoRegression, self).__init__()
+
+    def fit(self, y, X=None, fh=None):
+
+        self._is_fitted = False
+        self._set_fh(fh)
+
+        if isinstance(y, pd.DataFrame) and type(y.index) == pd.Int64Index:
+            y, X = _coerce_int_to_range_index(y, X)
+            y = check_series(y)
+        self._y = y
+
+        self._set_cutoff(y.index[-1])
+        self._fit(y=y, X=X, fh=fh)
+        self._is_fitted = True
+        return self
+
+    def _fit(self, y, X=None, fh=None):
+
+        self._forecaster = _VAR(
+            y, dates=self.dates, freq=self.freq, missing=self.missing
+        )
+        self._fitted_forecaster = self._forecaster.fit()
+        return self
+
+    def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        """
+        Make forecasts.
+
+        Parameters
+        ----------
+        fh : ForecastingHorizon
+            The forecasters horizon with the steps ahead to to predict.
+            Default is one-step ahead forecast,
+            i.e. np.array([1])
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored.
+        return_pred_int : bool, optional (default=False)
+        alpha : int or list, optional (default=0.95)
+
+        Returns
+        -------
+        y_pred : pd.Series
+            Returns series of predicted values.
+        """
+        self.check_is_fitted()
```

Comment:
this should not happen in `_predict`, it's now all in `predict`. Please fix.

Reply:
okay. removed